components/automate-deployment/pkg/majorupgradechecklist/v3.go

```diff
@@ -0,0 +1,323 @@
+package majorupgradechecklist
+
+import (
+	"fmt"
+
+	"github.com/chef/automate/components/automate-cli/pkg/status"
+	"github.com/chef/automate/components/automate-deployment/pkg/cli"
+	platform_config "github.com/chef/automate/lib/platform/config"
+	"github.com/pkg/errors"
+)
+
+const (
+	initMsg = `This is a Major upgrade. 
+========================
+
+  1) In this release Embedded PostgreSQL is upgraded to version 13.5 
+  2) This will need special care if you use Embedded PostgreSQL. 
+
+===== Your installation is using %s PostgreSQL =====
+
+  Please confirm this checklist that you have taken care of these steps 
+  before continuing with the Upgrade to version %s:
+`
+
+	downTimeError = "There will be a downtime while upgrading. Please prepare for down time and run the upgrade"
+
+	backupError = "Please take a backup and restart the upgrade process."
+
+	run_chef_automate_upgrade_status_cmd = `chef-automate upgrade status`
+	run_chef_automate_upgrade_status     = `Check the status of your upgrade using:  
+     $ ` + run_chef_automate_upgrade_status_cmd + `
+   This should return: Automate is up-to-date`
+
+	run_pg_data_migrate = `Migrate Data from PG 9.6 to PG 13.5 using this command:
+     $ ` + run_pg_data_migrate_cmd
+
+	run_pg_data_migrate_cmd = `chef-automate post-major-upgrade migrate --data=pg`
+
+	run_chef_automate_status_cmd = `chef-automate status`
+	run_chef_automate_status     = `Check all services are running using: 
+     $ chef-automate status`
+
+	run_pg_data_cleanup = `If you are sure all data is available in Upgraded Automate, then we can free up old PostgreSQL 9.6 Data by running: 
+     $ ` + run_pg_data_cleanup_cmd
+
+	run_pg_data_cleanup_cmd        = `chef-automate post-major-upgrade clear-data --data=PG`
```

Comment:
--data=PG are we sure this should be `PG` and not `pg`

Reply:
will accept both pg and PG